sh/koupler.sh

```diff
@@ -3,7 +3,7 @@
 BASEDIR=$(dirname $0)
 
 CLASSPATH=
-for i in `ls ./lib/*.jar`
+for i in `ls build/libs/*.jar`
```

Comment:
This gets packaged into the distribution file, where it looks in ./libs.  If we make this changed, we will have to adjust the distribution artifact.